core/store/prometheus.go

```diff
@@ -0,0 +1,38 @@
+package store
```

Comment:
Out of curiosity, why is this in store if it doesnt touch the DB?

Reply:
`store` is a little bit more loosely related to persistence than maybe a usual app. In our case that includes connections to the blockchain, which is why the ETH balance monitoring is in there.